def test_failure():
    assert 1 + 1 == 3